contact.php

```diff
@@ -15,7 +17,7 @@
 
         <div class="content">
         <h1  class="centerText"><u>Contact Us</u></h1>
-
+        <img src="images/heart_latte_art.jpg" alt="Latte art with heart shaped art" width="50%" >
```

Comment:
The **_heart_latte_art.jpg_**  has to be corrected to: **_heart_latte_art.jpeg_**!